dev/alchemist/DEAfrica_wofs_c2.yaml

```diff
@@ -0,0 +1,64 @@
+specification:
+  product: ls8_sr
+  measurements: ['blue', 'green', 'red', 'nir', 'swir_1', 'swir_2', 'pixel_quality']
+  #measurements: ['nbart_blue', 'nbart_green', 'nbart_red', 'nbart_nir', 'nbart_swir_1', 'nbart_swir_2', 'oa_fmask']
+  measurement_renames:
+    blue: nbart_blue
+    green: nbart_green
+    red: nbart_red
+    nir: nbart_nir
+    swir_1: nbart_swir_1
+    swir_2: nbart_swir_2
+    pixel_quality: fmask
+
+  aws_unsigned: False
+  transform: wofs.virtualproduct.WOfSClassifier
+  transform_url: 'https://github.com/GeoscienceAustralia/wofs/'
+  override_product_family: ard # The relationship of the source dataset to the newly derived dataset
+  basis: green
+
+  transform_args:
+    c2_scaling: True
+    dsm_path:  's3://deafrica-input-datasets/srtm_dem/srtm_africa.tif'
+    #dsm_path:  'https://deafrica-data.s3-us-west-2.amazonaws.com/ancillary/dem/srtm_africa.tif' #'s3://deafrica-data/ancillary/dem/srtm_africa.tif'
+
+output:
+  location: '/home/jovyan/Alchemist/' #'s3://deafrica-data-dev/wofs/new'
+  nodata: 1
+  dtype: uint8
+  explorer_url: https://explorer.dev.digitalearth.africa/products/ls8_c2l2/extents
+  preview_image_singleband:
+    measurement: water
+    lookup_table:
+      0: [150, 150, 110]   # dry
+      1: [0, 0, 0]   # nodata,
+      16: [119, 104, 87]   # terrain
+      32: [89, 88, 86]     # cloud_shadow
+      64: [216, 215, 214]  # cloud
+      80: [242, 220, 180]  # cloudy terrain
```

Comment:
Some are not here that are in the below.

Can you please try to keep the wording consistent too, be creative! Use emojis?